server/controllers/profileController.js

```diff
@@ -0,0 +1,70 @@
+const Profile = require("../models/ProfileModel");
+const mongoose = require("mongoose");
+
+const toObjectId = mongoose.Types.ObjectId;
+
+// create new profile using given arguments
+
+exports.createProfile = async (req, res, next) => {
+  try {
+    const { firstName, lasttName, phone, address, gender, description } =
+      req.body;
+    const data = {
+      userId: toObjectId(req.params.id) || "",
+      firstName: firstName.trim().toString() || "",
+      lasttName: lasttName.trim().toString() || "",
+      phone: phone.trim().toString() || "",
+      address: address.trim().toString() || "",
+      gender: gender.trim().toString() || "",
+      description: description.trim().toString() || "",
+    };
+    const newProfile = await new Profile(data);
```

Comment:
We don't need to `await` here as the profile creation does not return a promise :)